.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug report
+description: Create a report to help us improve
+labels: Bug
+
+body:
+- type: textarea
+  attributes:
+    label: Describe the bug
+    placeholder: A clear and concise description of what the bug is.
+  validations:
+    required: true
+- type: dropdown
+  attributes:
+    label: Area of issue
+    description: What component(s) of Open-Shell does this involve?
+    multiple: true
+    options:
+      - Start menu
```

Comment:
Maybe we can add `Taskbar` option too?
As we are changing the behavior of taskbar too and it may be not obvious to everyone that it is handled in `Start menu` component.

Reply:
Yes, that's a good idea for clarification. I will add that.